client/components/plans/plan-price/index.jsx

```diff
@@ -59,6 +59,12 @@ const PlanPrice = React.createClass( {
 			periodLabel = '';
 		} else if ( abtest( 'planPricing' ) === 'monthly' && plan.raw_price > 0 ) {
 			periodLabel = this.translate( 'per month, billed yearly' );
+			if ( plan.product_type === 'jetpack' ) {
+				//jetpack supports monthly plans
+				if ( plan.bill_period === 31 ) {
```

Comment:
A bit tricky to use 365 as a year and 31 as a month. I would propose to hide it behind constant, to avoid future discussions :D